manifests/module.pp

```diff
@@ -44,7 +47,7 @@
     require ::selinux::refpolicy_package
   }
 
-  if ($builder == 'simple' and $source_if != undef) {
+  if ($builder == 'simple' and ($source_if != undef or $content_if != undef)) {
```

Comment:
@lightoze  I like the more compact style now that there is more than one parameter, but it doesn't matter that much. Fix it if you like. :)